### ISSUE TITLE
Icon background after 14.13.5

### ISSUE THREAD

**Issue by lucaslacerdacl** [plantagoIT/strapi-drag-drop-content-type-plugin#53]
Hello everyone! I'm using this library and recently I update Strapi to version 14.13.6.
After open my admin I notice that the order rank button was not the same.

```
  "dependencies": {
    "@retikolo/drag-drop-content-types": "^1.3.9",
    "@strapi/strapi": "4.13.6"
  }
```

The button looks like this:

![Screenshot 2023-09-18 at 08 47 02](https://github.com/plantagoIT/strapi-drag-drop-content-type-plugin/assets/6237330/01855f51-aac2-4407-94e3-4adb7701c36c)

**Comment by lucaslacerdacl** [comment]
anyone?

**Comment by derschiw** [comment]
I will take a look, thanks for your report!

**Comment by derschiw** [comment]
I couldn't reproduce this with strapi version 4.13.7, maybe update the minor version or can you provide more information?

**Comment by arastorhuiev** [comment]
same problem
    "@retikolo/drag-drop-content-types": "^1.3.9",
    "@strapi/plugin-i18n": "4.13.7",
    "@strapi/plugin-users-permissions": "4.13.7",
    "@strapi/provider-email-mailgun": "^4.14.0",
    "@strapi/strapi": "4.13.7",
    "better-sqlite3": "8.0.1",
    "dotenv": "^16.3.1",
    "react-copy-to-clipboard": "^5.1.0"

**Comment by arastorhuiev** [comment]
on 14.3.5 - ok, everything above - problem actually

**Comment by lucaslacerdacl** [comment]
Hello guys! Well, you are correct @arastorhuiev , the problem rises after that release.

**Comment by lucaslacerdacl** [comment]
The button is working just fine, I guess is only style problem.

**Comment by lucaslacerdacl** [comment]
Is there some documentation about contribution? I would like to try fix it.

**Comment by troc274** [comment]
I have the same issue. Anyone know how to fix it? 
`
"dependencies": {
    "@retikolo/drag-drop-content-types": "^1.3.9",
    "@strapi/strapi": "4.13.7"
  }
`
![Screenshot 2023-10-07 at 9 28 44 PM](https://github.com/plantagoIT/strapi-drag-drop-content-type-plugin/assets/44805100/406079b3-1b14-462e-b4e5-114e27aeaa99)

**Comment by derschiw** [comment]
> Is there some documentation about contribution? I would like to try fix it.

It uses the Strapi Design System (https://design-system-git-main-strapijs.vercel.app/?path=/docs/design-system-components-iconbutton--docs). That's why I'm a little confused, how this could happen.

The code is here: 
https://github.com/plantagoIT/strapi-drag-drop-content-type-plugin/blob/5bdd4bcd0d2c7f5e2e866bdb24b0baf08cb4796b/admin/src/components/SortModal/index.js#L224

**Comment by derschiw** [comment]
I submitted an issue to the strapi design system. Consider upvoting to increase progress speed.

https://github.com/strapi/design-system/issues/1418

**Comment by derschiw** [comment]
Meanwhile I published a new version using another layout for the button. I will revert this as soon as Strapi fixes the other layout.

![image](https://github.com/plantagoIT/strapi-drag-drop-content-type-plugin/assets/37687705/04d79bf8-c7cb-44ef-9372-76f89c64e7cf)

**Comment by derschiw** [comment]
solved with #64
